Src/Plugins/SalModernization/SEC105_001.AllocatorValidator.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    public class AllocatorValidator : ValidatorBase
```

Comment:
>AllocatorValidator [](start = 17, length = 18)

do we need this?

Reply:
This was the dummy validator that I thought Michael was suggesting, is this not what you two were talking about?

---
In reply to: [605122134](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r605122134) [](ancestors = 605122134)

Reply:
we dont need this. Since when we build it will automatically generate the dll.
You will also have to bring the dll to the spam folder because it will "require" since u added in the json

---
In reply to: [605180704](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r605180704) [](ancestors = 605180704,605122134)

Reply:
Right, you mentioned that creating a "build targets" file like the one that exists in Security would move the DLL into the .spam folder, but I found that it wasn't doing that for anything and you said you wanted to take a look at it.  Would you like me to investigate instead?

---
In reply to: [605182112](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r605182112) [](ancestors = 605182112,605180704,605122134)

Reply:
i already did. check that link that i sent in the other comment. you will see the dll movement as well :)

---
In reply to: [605183649](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r605183649) [](ancestors = 605183649,605182112,605180704,605122134)

Reply:
Deleted, and proj file corrected

---
In reply to: [605185776](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r605185776) [](ancestors = 605185776,605183649,605182112,605180704,605122134)